src/pages/iouReportPropTypes.js

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types';
 
 export default PropTypes.shape({
     /** The report ID of the IOU */
-    reportID: PropTypes.string,
+    reportID: PropTypes.number,
```

Comment:
Confused I am seeing this somewhere, should this be a number or string?

 * @param {String} iouReportID

Reply:
We are transitioning to strings. So you'll find both right now. This particular one should be string though. Updated!